[LB] Get started (#2122)
cloudflare/cloudflare-docs: products/load-balancing/src/content/get-started/index.md

---

## Prerequisites

- **Multiple servers**, either physical or cloud-based.
- **Access to Load Balancing**, available as an add-on for any type of account.
- **Load balancer hostname**: The hostname for which the Cloudflare Load Balancer will manage traffic. The default hostname is the root hostname.

---

## Step 1 — Create a monitor

<MonitorDefinition/>

<details>
<summary>Create a monitor (dashboard)</summary>
<div>

<strong>Set up the monitor</strong>

<MonitorCreate/>

<strong>Prepare your servers</strong>

<MonitorPrepareServers/>

</div>
</details>

<details>
<summary>Create a monitor (API)</summary>
<div>

<strong>Set up the monitor</strong>

<MonitorCreateAPI/>

<strong>Prepare your servers</strong>

<MonitorPrepareServers/>

</div>
</details>

<MonitorExample/>

## Step 2 — Create an origin pool

<PoolDefinition/>

<details>
<summary>Create a pool (dashboard)</summary>
<div>

<PoolCreate/>

</div>

</details>

<details>
<summary>Create a pool (API)</summary>
<div>

<PoolCreateAPI/>

</div>

</details>

## Step 3 — Confirm pool health

> Before directing any traffic to your pools, you want to make sure that your pools and monitors are set up correctly. The status of your health check will be *unknown* until the results of the first check are available.

Reviewer: ```suggestion
Before directing any traffic to your pools, make sure that your pools and monitors are set up correctly. The status of your health check will be *unknown* until the results of the first check are available.
```

<details>
<summary>Confirm pool health (dashboard)</summary>
<div>

1. Navigate to **Traffic** > **Load Balancing**.
1. Click **Manage Pools**.
1. For pools and individual origins, review the values in the **Health** and **Origin Health** columns.

For more information on pool and origin health statuses, refer to [How a pool becomes unhealthy](understand-basics/health-details#how-a-pool-becomes-unhealthy).

</div>

</details>

<details>
<summary>Confirm pool health (API)</summary>
<div>

To fetch the latest health status of all pools, use the [List Pools](https://api.cloudflare.com/#account-load-balancer-pools-list-pools) command, paying attention to the `healthy` value for pools and origins.

For troubleshooting a specific pool's health, use the [Pool Health Details](https://api.cloudflare.com/#account-load-balancer-pools-pool-health-details) command.

</div>

</details>

If you notice that healthy pools are being marked unhealthy:

- Review [how origins and pools become unhealthy](/understand-basics/health-details).

> - Refer to our [Troubleshooting FAQ](https://support.cloudflare.com/hc/articles/4407016052493).

Reviewer: ```suggestion
- Refer to the [Troubleshooting FAQ](https://support.cloudflare.com/hc/articles/4407016052493).
```

## Step 4 — Create a load balancer on a test subdomain

<LBDefinition/>

<details>
<summary>Create a load balancer (dashboard)</summary>
<div>

<LBCreate/>

</div>

</details>

<details>
<summary>Create a load balancer (API)</summary>
<div>

<LBCreateAPI/>

</div>

</details>

### Route traffic to your load balancer

> Just as in the previous step, you want to make sure your load balancer is functioning as you expected before using it with live traffic.

Reviewer: ```suggestion
Just as in the previous step, make sure your load balancer is functioning as you expected before using it with live traffic.
```

For example, if you had `test.example.com` as a testing subdomain, you could either:

- Create a load balancer with a **Hostname** of `test.example.com`.
- Create a load balancer with a different **Hostname** (`lb.example.com`) and set up a CNAME record on `test.example.com` that points to `lb.example.com`.

Either option would use your load balancer to distribute requests going to `test.example.com`.

## Step 5 — Review load balancing analytics

> As you send sample requests to your test domain, review our [load balancing analytics](/reference/load-balancing-analytics) to make sure your load balancer is distributing requests like you were expecting.

Reviewer: ```suggestion
As you send sample requests to your test domain, review the [load balancing analytics](/reference/load-balancing-analytics) page to make sure your load balancer is distributing requests like you were expecting.
```
